systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfSharedST.java

```diff
@@ -183,6 +183,9 @@ private static Map<String, Object> generateTestCases(String kafkaVersion) {
 
             // skipping these configuration exceptions
             testCases.remove("ssl.cipher.suites");
+
+            // skipping these configuration
```

Comment:
can you just add a comment mentioning that from Kafka 4.x if ELR is enabled on the cluster we can't change this config anymore, so skipping it.